### PR TITLE
fix: Creating a default availability for org owner [CAL-2142]

### PR DESCRIPTION
**Pull request by leog**
## What does this PR do?

When a user creates an organization, they don't go through the profile onboarding, preventing them to have a default availability created for them.

This PR introduces the creation of a default availability when the owner account is created. Empty state was already there, just not accessible through UI.

## Type of change

  - [X] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

1. Create an organization
2. Go to the Availability page
3. See a default availability be created

> **Note**
> As you will not be able to delete the only availability to see the empty state, check it deleting availability from DB

## Mandatory Tasks

  - [ ] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

File: packages/trpc/server/routers/viewer/organizations/create.handler.ts [hunk 1]
```diff
@@ -77,13 +78,31 @@ export const createHandler = async ({ input }: CreateOptions) => {
 
   const emailDomain = adminEmail.split("@")[1];
 
+  const t = await getTranslation(ctx.user.locale ?? "en", "common");
+  const availability = getAvailabilityFromSchedule(DEFAULT_SCHEDULE);
```
Comment (leog) [inline packages/trpc/server/routers/viewer/organizations/create.handler.ts:82]: Luckily it is this easy to create a default availability 🌟

File: packages/trpc/server/routers/viewer/organizations/create.handler.ts [hunk 2]
```diff
@@ -77,13 +78,31 @@ export const createHandler = async ({ input }: CreateOptions) => {
...
+              createMany: {
+                data: availability.map((schedule) => ({
+                  days: schedule.days,
+                  startTime: schedule.startTime,
+                  endTime: schedule.endTime,
+                })),
+              },
+            },
+          },
+        },
```
Comment (leog) [inline packages/trpc/server/routers/viewer/organizations/create.handler.ts:105]: And then assigning it to the newly owner created user

File: apps/web/pages/availability/index.tsx [hunk 3]
```diff
@@ -95,6 +95,7 @@ export function AvailabilityList({ schedules }: RouterOutputs["viewer"]["availab
             Icon={Clock}
             headline={t("new_schedule_heading")}
             description={t("new_schedule_description")}
+            className="w-full"
```
Comment (leog) [inline apps/web/pages/availability/index.tsx:98]: Empty screen was not filling the whole space

**Review by sean-brydon — APPROVED** [review]
Havent tested but code looks good

**Review by emrysal — APPROVED** [review]

**Pull request merged**